0003-messaging-ipc/README.md

```diff
@@ -0,0 +1,98 @@
+---
+name: Messaging IPC
+shortname: 3/MIPC
+status: raw
+contributors: Andrew Kokhanovskyi <akokhanovskyi@cybervisiontech.com>
```

Comment:
> Honestly, I don't see a lot of value in specifying people / emails in the doc. Everyone can read the commit history and figure things out.

Well, @hunch7 just committed all the doc. He is the only contributor now, isn't he? The point is, the one who did commit (or even authored) isn't one, who really contributed to it.